documentation/book/ref-kafka-entity-operator.adoc

```diff
@@ -0,0 +1,146 @@
+
+== Topic Operator
+
+Topic Operator deployment can be configured using additional options inside the `topicOperator` object.
+Following options are supported:
+
+`watchedNamespace`::
+The {ProductPlatformName} namespace in which the topic operator watches for `KafkaTopics`.
+Default is the namespace where the Kafka cluster is deployed.
+
+`reconciliationIntervalSeconds`::
+The interval between periodic reconciliations in seconds. Default is 90.
+
+`zookeeperSessionTimeoutSeconds`::
+The Zookeeper session timeout in seconds. Default is 20 seconds.
+
+`topicMetadataMaxAttempts`::
+The number of attempts for getting topics metadata from Kafka.
+The time between each attempt is defined as an exponential back-off.
+You might want to increase this value when topic creation could take more time due to its many partitions or replicas. Default is `6`.
+
+`image`::
+The `image` property can be used to configure the container image which will be used.
+For more details about configuring custom container images, see xref:assembly-configuring-container-images-{context}[].
+
+`resources`::
+The `resources` property configures the amount of resources allocated to the Topic Operator
+For more details about resource request and limit configuration, see xref:assembly-resource-limits-and-requests-{context}[].
+
+`logging`::
+The `logging` property configures the logging of the Topic Operator
+For more details about logging configuration, see xref:assembly-logging-{context}[].
+
+.Example of Topic Operator configuration
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+  zookeeper:
+    # ...
+  entityOperator:
+    # ...
+    topicOperator:
+      watchedNamespace: my-topic-namespace
+      reconciliationIntervalSeconds: 60
+    # ...
+----
+
+== User Operator
+
+User Operator deployment can be configured using additional options inside the `userOperator` object.
+Following options are supported:
+
+`watchedNamespace`::
+The {ProductPlatformName} namespace in which the topic operator watches for `KafkaUsers`.
+Default is the namespace where the Kafka cluster is deployed.
+
+`reconciliationIntervalSeconds`::
+The interval between periodic reconciliations in seconds. Default is 120.
+
+`zookeeperSessionTimeoutSeconds`::
+The Zookeeper session timeout in seconds. Default is 6 seconds.
+
+`image`::
+The `image` property can be used to configure the container image which will be used.
+For more details about configuring custom container images, vseesit xref:assembly-configuring-container-images-{context}[].
+
+`resources`::
+The `resources` property configures the amount of resources allocated to the User Operator
+For more details about resource request and limit configuration, see xref:assembly-resource-limits-and-requests-{context}[].
+
+`logging`::
+The `logging` property configures the logging of the User Operator
```

Comment:
Add period at the end of the sentence: to the User Operator > to the User Operator.